index.html

```diff
@@ -601,18 +609,51 @@ <h3>Prohibited States and Properties</h3>
 				<p class="note">A host language attribute with the appropriate <a href="#implicit_semantics">implicit WAI-ARIA semantic</a> would also prohibit a state or property in this section. </p>
 			</section>
 		<section id="mustContain">
-			<h3>Required Owned Elements</h3>
-			<p>Any <a>element</a> that will be [=ARIA/owned=] by the element with this <a>role</a>. For example, an element with the role <rref>list</rref> will own at least one element with the role <rref>listitem</rref>.</p>
-			<p>When multiple roles are specified as <em>required owned elements</em> for a role, at least one instance of one required [=ARIA/owned=] element is expected. This specification does <em>not</em> require an instance of each of the listed owned roles. For example, a <code>menu</code> should have at least one instance of a <code>menuitem</code>, <code>menuitemcheckbox</code>, <em>or</em> <code>menuitemradio</code>. The <code>menu</code> role does not require one instance of each. </p>
-			<p>There might be times that required [=ARIA/owned=] elements are missing, for example, while editing or while loading a data set. When a widget is missing <em>required owned elements</em> due to script execution or loading, authors MUST mark a containing element with <sref>aria-busy</sref> equal to <code>true</code>. For example, until a page is fully initialized and complete, an author could mark the document element as busy.</p>
-			<p class="note">A role that has 'required owned elements' does not imply the reverse relationship. While processing of a role might be incomplete without elements of given roles present as descendants, elements with roles in this list do not always have to be found within elements of the given role. See <a href="#scope">required context role</a> for requirements about the context where elements of a given role will be contained.</p>
-			<p class="note">An element with a <a href="#subclassroles">subclass role</a> of the 'required owned element' does not fulfill this requirement. For example, the <rref>listbox</rref> role requires ownership of an element using the <rref>option</rref> or <rref>group</rref> role. Although the <rref>group</rref> role is the superclass of <rref>row</rref>, adding an [=ARIA/owned=] element with a role of <rref>row</rref> will not fulfill the requirement that <rref>listbox</rref> owns an <rref>option</rref> or a <rref>group</rref>.</p>
+			<h3>Allowed Accessibility Child Roles</h3>
+			<p>A list of roles which may be allowed on an <a>accessibility child</a> (simplified as "child") of the element with this <a>role</a>. For example, an element with the role <rref>list</rref> MAY own child elements with the role <rref>listitem</rref>, but MUST NOT own elements with the role <rref>option</rref>.</p>
```

Comment:
Is this first may an normative MAY? Seems like yes actually
```suggestion
			<p>A list of roles which MAY be allowed on an <a>accessibility child</a> (simplified as "child") of the element with this <a>role</a>. For example, an element with the role <rref>list</rref> MAY own child elements with the role <rref>listitem</rref>, but MUST NOT own elements with the role <rref>option</rref>.</p>
```

or replace with "A list of roles that are allowed"

Reply:
Actually, I think we'd need an actor (Author or User Agent) in each of the RFC-2119 normative statements, otherwise they are ambiguous and unenforceable with author warnings or automated browser tests.

Reply:
```suggestion
			<p>A list of roles which are allowed on an <a>accessibility child</a> (simplified as "child") of the element with this <a>role</a>. For example, an element with the role <rref>list</rref> can own child elements with the role <rref>listitem</rref>, but cannot own elements with the role <rref>option</rref>.</p>
```

Reply:
ok I see, @smhigley I thnk you can just commit @cookiecrook suggestiong above.